packages/@aws-cdk/aws-glue-alpha/lib/job-executable.ts

```diff
@@ -150,6 +160,12 @@ interface PythonExecutableProps {
 }
 
 interface SharedJobExecutableProps {
+  /**
+   * Runtime.
```

Comment:
stronger docs needed. need to know when is runtime required, etc

Reply:
Added bit more information.
Our public doc does not have much information about this property, but I tried to add as much as possible.